include/glow/Base/Tensor.h

```diff
@@ -672,10 +656,12 @@ template <class ElemTy> class Handle final {
   /// the value that is stored at coordinate {d_0, d_1, ... d_n} in the new
   /// tensor at {d_0 + O_0, d_1 + O_1, ... d_n + O_n}, where O is the offset
   /// vector. The tensors must be of the right dimensions.
-  void insertTensors(Handle<ElemTy> &slice, llvm::ArrayRef<size_t> offset) {
+  void insertTensors(Handle<ElemTy> &slice, llvm::ArrayRef<size_t> offset,
```

Comment:
Please add doxygen comments that describe the two new parameters.

Reply:
Done.